pkg/repository/provider/unified_repo.go

```diff
@@ -105,22 +111,124 @@ func (urp *unifiedRepoProvider) InitRepo(ctx context.Context, param RepoParam) e
 }
 
 func (urp *unifiedRepoProvider) ConnectToRepo(ctx context.Context, param RepoParam) error {
-	///TODO
+	log := urp.log.WithFields(logrus.Fields{
+		"BSL name": param.BackupLocation.Name,
+		"BSL UID":  param.BackupLocation.UID,
+	})
```

Comment:
All the member function in unifiedRepoProvider has the same log fields 'BSL name', 'BSL UID'. 
cloud set the field in unifiedRepoProvider initialization?

Reply:
The BSL is a param of the specific function calls, it also means, the BSL could be different for each function call.
Therefore, we don't have the BSL information at the time of the initialization.